### PR TITLE
[Term Entry] Semantic Technologies / SPARQL

### PR DESCRIPTION
**Pull request by noahpgordon**
### Description

Added **sparql.md** within General / Semantic Technologies. As described in the concept entry on Semantic Technologies, SPARQL is one of the three fundamental standards within the suite of semantic technologies. This entry completes the term entries for these three standards (with the existing rdf.md and owl.md).

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/general/concepts/semantic-technologies/terms/sparql/sparql.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+| ?title |
+|:-:|
+| "Pachinko" |
+| "The Mosquito Coast" |
+| "A Gesture Life" |
+
+The special character `*` can be used to return the values matching all variables in the query pattern:
+
+```
```
Comment (Sriparno08) [inline content/general/concepts/semantic-technologies/terms/sparql/sparql.md:95]:
````suggestion
```sparql
````

File: content/general/concepts/semantic-technologies/terms/sparql/sparql.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+This results in the output:
+
+|?x|?y|
+|:-:|:-:|
+|doc:authoredBy|https://en.wikipedia.org/wiki/Chang-Rae_Lee|
+|doc:title|"A Gesture Life"|
+|doc:soldCopies|87740|
+
+### `ASK`
```
Comment (Sriparno08) [inline content/general/concepts/semantic-technologies/terms/sparql/sparql.md:111]:
```suggestion
### ASK
```

**Review by Sriparno08 — COMMENTED** [review]
Hey @noahpgordon, thanks for contributing to Codecademy!

I've listed all the modifications that you need to make in your entry below:

**Comment by noahpgordon** [comment]
@Sriparno08 I committed the above suggestions.

**Comment by Sriparno08** [comment]
Good work, @noahpgordon.

The thing is that, you've committed only the visible suggestions. There are also suggestions that are automatically hidden. So, please go through the suggestions, click on 'Load more...' every time you see 'X hidden conversations' and commit all of them.

File: content/general/concepts/semantic-technologies/terms/sparql/sparql.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+}
+```
+
+This results in the output:
+
+| ?title |
+|:-:|
+| "Pachinko" |
+| "The Mosquito Coast" |
+| "A Gesture Life" |
```
[the remark this answers is not in the document]
Comment (noahpgordon) [reply content/general/concepts/semantic-technologies/terms/sparql/sparql.md:92]: I believe putting these within a pseudo code block will prevent them from being formatted as a table.

**Comment by noahpgordon** [comment]
@Sriparno08 Whoops. I committed the hiddens, with the exception of two that I think might mess with the formatting of tables.

**Review by Sriparno08 — APPROVED** [review]
Great job, @noahpgordon!

This looks good for a second review 👍

**Comment by noahpgordon** [comment]
@avdhoottt Is this going to be reviewed by you or someone else any time soon? Original PR was over 6 months ago and the first review was completed over 4 months ago.

**Review by avdhoottt — APPROVED** [review]
LGTM!

**Pull request merged**